src/components/Editable.tsx

```diff
@@ -581,6 +581,7 @@ const Editable = ({
 
   return (
     <ContentEditable
+      data-rank={rank}
```

Comment:
Unused?

Reply:
Yes, this has been removed now.

Reply:
`data-rank={rank}` is still in the diff and appears to be unused.